pkg/gimletd/worker/branchDeleteEvent.go

```diff
@@ -67,16 +67,32 @@ func (r *BranchDeleteEventWorker) Run() {
 					continue
 				}
 
+				branchesWithManifests := map[string][]*dx.Manifest{}
+				branches := commonGit.BranchList(repo)
+				for _, branch := range branches {
+					manifests, err := r.extractManifestsFromBranch(repo, branch)
+					if err != nil {
+						logrus.Warnf("could not extract manifests: %s", err)
+						continue
+					}
+
+					if branchesWithManifests[branch] == nil {
+						branchesWithManifests[branch] = []*dx.Manifest{}
+					}
+
+					branchesWithManifests[branch] = manifests
+				}
+
 				deletedBranches, err := r.detectDeletedBranches(repo)
 				if err != nil {
 					logrus.Warnf("could not detect deleted branches in %s: %s", repoPath, err)
 					os.RemoveAll(repoPath)
 					continue
 				}
 				for _, deletedBranch := range deletedBranches {
-					manifests, err := r.extractManifestsFromBranch(repo, deletedBranch)
-					if err != nil {
-						logrus.Warnf("could not extract manifests: %s", err)
+					manifests := branchesWithManifests[deletedBranch]
+					if len(manifests) == 0 {
+						logrus.Warn("could not extract manifests")
```

Comment:
Drop this warning.

It is perfectly fine to not have any gimlet manifest on a branch.

This will add noise to the logs